kubetest2/kubetest2-gce/deployer/deployer.go

```diff
@@ -60,14 +65,12 @@ func (d *deployer) Provider() string {
 	return Name
 }
 
-func (d *deployer) Up() error { return nil }
-
 func (d *deployer) IsUp() (up bool, err error) { return false, nil }
 
 func (d *deployer) DumpClusterLogs() error { return nil }
 
-func (d *deployer) TestSetup() error { return nil }
-
-func (d *deployer) Kubeconfig() (string, error) { return "", nil }
+func (d *deployer) Kubeconfig() (string, error) {
+	return d.kubeconfigPath, nil
```

Comment:
maybe return an error if this kubeconfig was not populated

Reply:
I like that. Done.